src/transformers/models/time_series_transformer/modeling_time_series_transformer.py

```diff
@@ -918,28 +1010,32 @@ def _set_gradient_checkpointing(self, module, value=False):
 TIME_SERIES_TRANSFORMER_INPUTS_DOCSTRING = r"""
     Args:
         past_values (`torch.FloatTensor` of shape `(batch_size, sequence_length)`):
-            Past values of the time series, that serve as context in order to predict the future. These values may
-            contain lags, i.e. additional values from the past which are added in order to serve as "extra context".
-            The `past_values` is what the Transformer encoder gets as input (with optional additional features, such as
-            `static_categorical_features`, `static_real_features`, `past_time_features`).
+            Past values of the time series, that serve as context in order to predict the future. The sequence size of
+            this tensor must be larger than the `context_length` of the model, since the model will use the larger size
```

Comment:
Hmm what is you don't want to use lags? Isn't the sequence size then equal to `context_length`?

Reply:
it then has to be context_length + 1